src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -312,10 +312,13 @@ private double getLeftDistance() {
    * @see #getLeftDistance()
    * @since 1/30/2023
    */
-  private double getRightDistance() {
+  public double getRightDistance() {
     return nativeUnitsToDistanceMeters(rightMaster.getSelectedSensorPosition());
   }
 
+  public double getAverageDistance() {
+    return (getLeftDistance() + getRightDistance()) / 2;
+  }
```

Comment:
We should ideally have javadoc comments for getAverageDistance

Reply:
ok